extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/menu/DependenciesProcessor.java

```diff
@@ -84,7 +84,7 @@ void createBuildTimeActions(BuildProducer<BuildTimeActionBuildItem> buildTimeAct
     private boolean isEnabled() {
         var value = ConfigProvider.getConfig().getConfigValue("quarkus.bootstrap.incubating-model-resolver");
         // if it's not false and if it's false it doesn't come from the default value
-        return value == null || !"false".equals(value.getValue()) || "default values".equals(value.getSourceName());
+        return value == null || !"false".equals(value.getValue()) || "DefaultValuesConfigSource".equals(value.getSourceName());
```

Comment:
@radcortez could we have constants for these names because things are very brittle this way as demonstrated here?

Reply:
Or something like `value.isDefaultValue()`, i guess it'd be more complicated though